ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1165,7 +1170,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
+                                <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                                <Button Click="HUDAndVisuals_Reset" Width="22" Height="22" ToolTip="{DynamicResource ServerSettings_ResetSectionTooltip}" Margin="20,0,0,0">
```

Comment:
I will have a look at the command definitions in the RCON window. Was just trying to keep it the same so the coding was consistent.